### PR TITLE
Overflow of Request a free instructor Account outside of container #6879

### LINKED ISSUE #6879: Overflow of "Request a free instructor Account" outside of container
In the mobile view, we can see clearly there is an overflow of content/button as seen in this image.
And there should also be a little space between both the buttons.

screenshot from 2017-03-20 03-22-04

### PR DESCRIPTION
**Pull request by amarlearning**
Fixes #6879 

**Outline of Solution**

> Gave margin-bottom to the div surrounding those buttons

**Before:**

screenshot from 2017-03-20 03-22-04

**After:**
screenshot from 2017-03-20 18-56-55

### CODE REVIEW

**Comment by amarlearning** [comment]
@xpdavid Please review

**Comment by li-kai** [comment]
This is not the correct approach to this problem. Take a look at the `#raisedEdge` selector. The right approach would be to move the `top` selector to only be applied on screens smaller than 768px.

**Comment by amarlearning** [comment]
@li-kai What about the `spacing` between two buttons?

**Comment by li-kai** [comment]
@amarlearning Good eye, I didn't even noticed that change. Could you add `.margin-bottom-10px` to the video tour button?

**Comment by amarlearning** [comment]
@li-kai If i add `.margin-bottom-10px` it would work on all screen resolution and was not looking good as it creates way too extra space between buttons and that image on `786 < screen < 1366`. But I want them to work only on screens smaller than 768px.

Let me know your thoughts on this!

**Comment by li-kai** [comment]
@amarlearning I've added that, the `.margin-bottom-10px` to video tour button. The effect only triggers >768px.

767px
screenshot 2017-03-22 09 49 38
768px
screenshot 2017-03-22 09 50 05

**Comment by amarlearning** [comment]
@li-kai 
screenshot from 2017-03-22 18-19-44

File: src/main/webapp/stylesheets/teammatesCommon.css [hunk 1]
```diff
@@ -613,6 +613,12 @@ Apply to thead to make headers not bold.
     top: -2em;
 }
 
+@media screen and (max-width: 768px) {
+    #raisedEdge {
+        top: 0;
```
Comment (li-kai) [inline src/main/webapp/stylesheets/teammatesCommon.css]: I'd put `top: -2em;` with `(min-width:...` and remove the one on the top.

**Pull request merged**